src/server/game/Entities/Object/Object.cpp

```diff
@@ -1193,6 +1193,35 @@ bool WorldObject::IsWithinLOSInMap(const WorldObject* obj) const
     return IsWithinLOS(ox, oy, oz);
 }
 
+bool WorldObject::IsWithinBox(const Position& center, float xradius, float yradius, float zradius) const
```

Comment:
Since this method does not access any WorldObject members you can move it to Position class